enterprise/urls.py

```diff
@@ -15,6 +17,10 @@
         name='grant_data_sharing_permissions'
     ),
     url(
+        r'^enterprise/course/{}/$'.format(settings.COURSE_ID_PATTERN),
```

Comment:
@douglashall I like your suggested url pattern. As for us both `enterprise_uuid` and `course_id` are required.